### ISSUE TITLE
Make title view compatible with iPhone XR screen

### ISSUE THREAD

**Issue by GillianYue** [GillianYue/Bubbla#9]
It supposedly is but was not when I tested it. Oh well.

**Comment by GillianYue** [comment]
Since we're in world-space mode now (no longer using screen overlay for canvas), might need to write a script to initialize canvas size on start based off of screen resolution.

**Comment by GillianYue** [comment]
So I'm working on that canvas initialize scaling script right now - the scaling functions are located statically in the Global class. Unity has a messy Rect Transform scaling system, but mostly it's about deciding whether the anchors are 0-0, 0-0 (in which case the sizes will be fixed), or 0-1, 0-1 (in which case the sizes will be 100% dependent on the parent). After the anchors are set, the actual rect size can be set accordingly by modifying either the sizeDelta value (see [doc](https://docs.unity3d.com/ScriptReference/RectTransform-sizeDelta.html) sizeDelta means different things under different contexts) or setting offset Max (upper right corner of the rect relative to upper right anchor position) and offset Min (lower left).

**Comment by GillianYue** [comment]
Update:

I've figured out the way to make this work. The main canvas will be fixed size from now on (always 640*1136). The main camera will adjust its orthographic size to be able to see just the main canvas + the additional black space in the y-direction. 

Before, I modified the main canvas dimension based on the current resolution. So if the game is running on XR the main canvas will be 828*1792, and so on. But this creates a problem: Even if we force the aspect ratio to stay the same by creating black spaces on top and bottom of the screen, the physical width is different, and the elements under the main canvas will have to adjust to that, and keeping all of them consistent is going to be an annoying process.

So the solution is to have literally the same canvas no matter what, and let the main camera adjust its "distance to the game" based on the phone resolution. The same 640*1136 will be observed, and the extra height in y will be covered in black space.
